cmd/podman/run.go

```diff
@@ -54,6 +54,10 @@ func runCmd(c *cli.Context) error {
 	if err != nil {
 		return err
 	}
+	useImageVolumes := true
```

Comment:
Does this work?
useImageVolumes  =  (createConfig.ImageVolumeType == "ignore" )

Reply:
Tmpfs should also be false, should only set in libpod if we're creating sources for the volumes

Reply:
fixed